icons/kiwix.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Kiwix</title><path d="M18.298 15.359c-.663-2.037-2.906-3.167-5.014-2.526-2.107.64-3.277 2.809-2.614 4.845l.01.026H8.784a.814.814 0 0 1-.786-.561l-1.57-4.983.268-.138c3.723-1.511 7.707 1.185 10.48-1.403l.017-.016c.344-.303.687-.608 1.703.003 1.092.66 3.502 2.388 4.593 5.338 0 0 .663-.227.48-1.012-.184-.785-1.527-3.045-4.274-5.049a30.27 30.27 0 0 0-1.332-.92c.547-2.266-2.302-5.003-5.082-3.715-1.995-1.813-4.222-2.556-6.268-2.526-5.422.072-9.606 5.532-5.106 10.742l.01.01c.068.079.14.158.211.24.292.338.486.637.653 1.166l1.339 4.25h-.472a.778.778 0 0 0-.792.766h1.801l3.025 1.383a.754.754 0 0 0-.377-1.019l-.795-.36h2.28A.778.778 0 0 0 8 19.133H5.932a.814.814 0 0 1-.785-.562l-1.064-3.374c-.266-.886.343-1.465.914-1.534.52-.062.785.263 1.03 1.052l.945 2.992h-.473a.778.778 0 0 0-.792.765H7.51l3.025 1.383a.754.754 0 0 0-.377-1.018l-.795-.361h1.675c.922 1.52 2.839 2.283 4.65 1.73 2.105-.643 3.274-2.811 2.611-4.848zM15.256 6.67c.666 0 1.2.52 1.2 1.16 0 .204-.058.397-.153.565a.759.759 0 0 0-.769-.726.755.755 0 0 0-.768.742.73.73 0 0 0 .262.559 1.169 1.169 0 0 1-.976-1.14c0-.637.538-1.16 1.204-1.16z"/></svg>
```

Comment:
When comparing with https://wiki.kiwix.org/wiki/Logo#/media/File:Kiwix_logo_v3.svg I see the following differences (original in red, yours in black):
![image](https://github.com/user-attachments/assets/225c75a6-96db-41b0-8b7d-0e5f3faf0cc1)